frontend/src/lib/components/ui/heatmap/heatmap.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+  import {
+    HeatmapChart,
+    ScaleTypes,
+    type AreaChartOptions,
+    type HeatmapChartOptions,
+  } from '@carbon/charts-svelte';
+  import '@carbon/charts-svelte/styles.css';
```

Comment:
This was removed from the other chart components

Reply:
Fixed